src/metrics/prometheus.go

```diff
@@ -11,18 +11,25 @@ import (
 	"github.com/prometheus/client_golang/prometheus/push"
 	"github.com/prometheus/common/expfmt"
 
+	"github.com/thought-machine/please/src/cli"
 	"github.com/thought-machine/please/src/cli/logging"
-	"github.com/thought-machine/please/src/core"
+	"github.com/thought-machine/please/src/version"
 )
 
 var log = logging.Log
 
 var registerer = prometheus.WrapRegistererWith(prometheus.Labels{
-	"version": core.PleaseVersion,
+	"version": version.PleaseVersion,
 }, prometheus.DefaultRegisterer)
 
+type Config struct {
+	PrometheusGatewayURL string       `help:"The gateway URL to push prometheus updates to."`
```

Comment:
```suggestion
	PrometheusGatewayURL string       `help:"The aggregation gateway URL to push prometheus updates to."`
```
This should specifically be an aggregation gateway, sending them to a standard push gateway would be worthless. So may as well clarify that slightly here.